node/flatpak_node_generator/providers/special.py

```diff
@@ -328,7 +328,12 @@ async def _handle_playwright(self, package: Package) -> None:
             revision = int(browser['revision'])
 
             if name == 'chromium':
-                if revision < 792639:
+                # Revision number scheme was changed from Chromium version to incrementing
+                # integers above 1000; now it's same as for other browsers
+                if (
+                    SemVer.parse(package.version) < SemVer.parse('1.21.0')
+                    and revision < 792639
```

Comment:
small nit: I think 792639 is the Chromium *revision*, not just the version? Maybe it's just my tired brain, but I spent a very hot minute trying to figure out what on earth Chromium version 792639 is supposed to be :sweat_smile:

Reply:
Right, fixed the comment.